<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionMenu</class>
 <widget class="QDialog" name="ConnectionMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ConnectionWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#ConnectionMenu {
  border-image: url(:/img/connectionMenu.jpg) 0 0 0 0 stretch stretch;
  background-color: #0f1114;
}

QLabel#lblPort, QLabel#lblIp {
  color: #FFD700;
  font-weight: 700;
  font-size: 12pt;
  letter-spacing: 1px;
}

QLineEdit {
  background: #23262e;
  color: #e8eaed;
  border: 2px solid #3a3f4a;
  border-radius: 6px;
  padding: 10px 12px;
  font-size: 11pt;
  selection-background-color: #0d7377;
}

QLineEdit:focus {
  border: 2px solid #4a7c9e;
  background: #2a2d36;
}

QPushButton {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                              stop:0 #2d3139, stop:1 #23262e);
  color: #e8eaed;
  border: 2px solid #3a3f4a;
  border-radius: 8px;
  padding: 12px 18px;
  font-weight: 600;
  font-size: 12pt;
}

QPushButton:hover {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                              stop:0 #3a3f4a, stop:1 #2d3139);
  border: 2px solid #4a5060;
  color: #ffffff;
}

QPushButton:pressed {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                              stop:0 #1d2027, stop:1 #23262e);
}

QPushButton#btnConnect {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                              stop:0 #0d7377, stop:1 #0a5a5e);
  font-weight: bold;
}

QPushButton#btnConnect:hover {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                              stop:0 #14a5a5, stop:1 #0d7377);
}

QPushButton#btnExit {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                              stop:0 #6b3030, stop:1 #522424);
}

QPushButton#btnExit:hover {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                              stop:0 #8b4040, stop:1 #6b3030);
}
   </string>
  </property>
  <layout class="QVBoxLayout" name="root">
   <property name="leftMargin">
    <number>24</number>
   </property>
   <property name="topMargin">
    <number>24</number>
   </property>
   <property name="rightMargin">
    <number>24</number>
   </property>
   <property name="bottomMargin">
    <number>24</number>
   </property>
   <item>
    <spacer name="topSpace">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QFrame" name="panel">
     <property name="maximumSize">
      <size>
       <width>320</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="col">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="lblPort">
        <property name="text">
         <string>PORT</string>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="linePort">
        <property name="text">
         <string>8080</string>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblIp">
        <property name="text">
         <string>IP ADDRESS</string>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineHost">
        <property name="text">
         <string>localhost</string>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnConnect">
        <property name="text">
         <string>Connect</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExit">
        <property name="text">
         <string>Exit</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="bottomSpace">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <connections/>
</ui>
